src/components/duet-date-picker/date-picker-month.tsx

```diff
@@ -77,6 +85,7 @@ export const DatePickerMonth: FunctionalComponent<DatePickerMonthProps> = ({
       <tbody>
         {chunk(days, 7).map(week => (
           <tr class="duet-date__row">
+            {weekNumbers && <td class="duet-date__week">{weekOfYear(week[0])}</td>}
```

Comment:
I think you need `scope="row" role="rowheader"` here to properly indicate this is a header for the row.